unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -341,6 +341,21 @@ public void OnPressEsc()
         worldChatWindowHud.view.ActivatePreview();
     }
 
+    private void OnPressL()
```

Comment:
I'd change the name of this method to `OnToggleInputPress` or similar, because in the future we may change the key used to toggle this.
Also, I'm not sure why this is here if we have all the input detection logic in InputController, maybe `OnToggleInputPress` is not good enough to depict what the method does, can you think of a better method name?

Reply:
You are right, I have updated the name of the next methods:

- `OnPressL` to `OnFriendsToggleInputPress`.
- `OnPressReturn` to `OnWorldChatToggleInputPress`.
- `OnPressEsc` to `OnCloseWindowToggleInputPress`.

I think that makes it much clearer.

Note: These methods implements the specific logic (for this controller) that will be executed when each InputAction_Trigger (from InputController) be triggered. That is why they are here.

Thanks!